AutoHotkey/Numpad Hotkeys.ahk

```diff
@@ -23,33 +24,46 @@ NumpadDel:: Alt ; there are some issues with Alt.
 NumpadClear & Esc:: Send {blind}^w
 NumpadClear & NumpadPgUp:: Send {blind}^t
 NumpadClear & Tab:: Send {blind}^{tab}
-; this is disabled bc using Numpad0 in hotkeys messes with the hotstrings.
+; these are disabled bc using Numpad0 in hotkeys messes with the hotstrings.
 ; Numpad0 & Tab:: Send +{Tab} ; shift tab when number mode is on.
 ; Numpad0::Numpad0 ; this allows you to use hotkey and keep the button
 NumpadClear & NumpadAdd:: Send {blind}^{+}
 NumpadClear & NumpadSub:: Send {blind}^{-}
 
 NumpadDel & NumpadLeft:: Send !{NumpadLeft} ; page left
 NumpadDel & NumpadRight:: Send !{NumpadRight} ; page right
-
+; text editing commands
 NumpadClear & NumpadRight:: Send, {blind}^{Right} ; word right
 NumpadClear & NumpadLeft:: Send, {blind}^{Left} ; word left
 
-; text editing commands
 NumpadClear & (:: Send ^z
 NumpadClear & ):: Send ^y
 NumpadClear & $:: Send {blind}^s
 NumpadClear & =:: Send ^x
 NumpadClear & NumpadDiv:: Send {blind}^c
 NumpadClear & NumpadMult:: Send {blind}^v
 NumpadClear & NumpadPgDn:: Send ^a
-NumpadClear & NumpadHome:: Send ^{Home}
+NumpadClear & NumpadHome:: Send ^{Home} ; these shouldn't be needed if Clr is Ctrl but fo some reason it is.
 NumpadClear & NumpadEnd:: Send ^{End}
 
 ; windows
 NumpadDel & Esc:: Send !{F4} ; close the window
 NumpadDel & Tab:: Send {blind}!{Tab} ; switch window. note: does not hold alt, must use Ctrl
 
+; make app button toggle calculator instead of open new
+~Launch_App2::
+if WinExist("Calculator","Calculator") {
+    loop 2 {
+      Sleep 200
+      WinClose , Calculator , , 2
+    }
+}
+return

```

Comment:
calculator will open if there is no calculator open already. `~` means button will still be sent no matter the hotkey.